release_notes/Service_Fabric_ReleaseNotes_90.md

```diff
@@ -0,0 +1,153 @@
+| 2. | **Feature** | [**General Availability**] Support for Ubuntu 20.04 | **Brief Description**: Service Fabric is available for use on Ubuntu 20.04 LTS based compute.<br>**Impact**:Service Fabric now supports creation of clusters that utilize Ubuntu 20.04 LTS. All current Linux-based features are fully supported. It is encouraged, after thorough validation of your workload, to upgrade clusters to use 20.04 LTS. Existing 16.04 LTS and 18.04 LTS clusters can directly be upgraded to 20.04.<br>**Documentation Reference**: Read more about it at -<ul><li>[Deploy a Linux Service Fabric cluster into an Azure virtual network](https://docs.microsoft.com/en-us/azure/service-fabric/service-fabric-tutorial-create-vnet-and-linux-cluster)</li><li>[Set up a Container based local cluster](https://docs.microsoft.com/en-us/azure/service-fabric/service-fabric-get-started-linux?tabs=sdksetupubuntu%2Clocalclusteroneboxcontainer#set-up-a-local-cluster)</li><li>[Upgrading Linux OS for Azure Service Fabric](https://docs.microsoft.com/en-us/azure/service-fabric/service-fabric-cluster-upgrade-os)</li><li>[Ubuntu 20.04 Service Fabric cluster - Sample Templates](https://github.com/Azure-Samples/service-fabric-cluster-templates/tree/master/7-VM-Ubuntu-2004-3-NodeTypes-Secure)</li><li>Also see feature *Support creation of End-to-End Developer Experience for Linux development on Windows using WSL2* listed below</li></ul> |
+| 3. | **Feature** | [**General Availability**] Support for Multi-AZ VMSS | **Brief Description**: AZ Spanning Virtual Machine Scale Sets (VMSS) has been in preview for a year becasue we did not have the ability to orchestrate VMSS updates, one AZ at a time until now.<br>We can have the ability to drive VMSS updates, sequentially 1 zone at a time by setting VMSSZonalUpgradeMode = "Hierarchical". This value can be changed at anytime.<br>**Documentation Reference**: Read more about it at -<ul><li>[Deploy an Azure Service Fabric cluster across Availability Zones](https://docs.microsoft.com/en-us/azure/service-fabric/service-fabric-cross-availability-zones)</li></ul> |
```

Comment:
AZ Spanning Virtual Machine Scale Sets (VMSS) has been in preview for a year becasue we did not have the ability to orchestrate VMSS updates, one AZ at a time until now.  

I would recommend phrasing as: 
AZ Spanning Virtual Machine Scale Sets (VMSS) has been in preview and is now entering general availability. 

We can have the ability to drive VMSS updates, sequentially 1 zone at a time by setting VMSSZonalUpgradeMode = "Hierarchical". This value can be changed at anytime.

-> 
VMSS upgrades can now be configured to occur sequentially 1 zone at a time by setting VMSSZonalUpgradeMode = "Hierarchical". This value can be updated on the VMSS at anytime after creation.